.github/workflows/pnpm_docker_image_from_config.yml

```
@@ -0,0 +1,264 @@
+                default: ""
+                description: root dir for pnpm monorepo
+            run_poeditor:
+                required: false
+                default: false
+                type: boolean
+            poeditor_get_translations_filename:
+                required: false
+                default: getTranslations.py
+                type: string
+                description: name of the getTranslations script. Defaults to getTranslations.py.
+            run_build:
+                required: false
+                default: true
+                type: boolean
+            dirs_for_docker_build:
+                required: true
+                type: string
+                description: Space-separated list of files or glob pattern to include in docker build context. E.g. 'public .next' for .next builds or 'build' for other web builds.
+            runs_on:
+                required: false
+                default: '["ubuntu-latest"]'
+                type: string
+                description: Set to '["self-hosted", "linux"]' if needed.
+            dockerfile_name:
+                required: false
+                default: 'Dockerfile'
+                type: string
+                description: Name of the Dockerfile, defaults to 'Dockerfile'
+            docker_build_args:
+                required: false
+                type: string
+                default: ''
+                description: docker build args in the form 'KEY1=value1,KEY2=value2'
+            use_custom_registry:
+                type: boolean
+                default: false
+                description: Use custom npm registry, uses default 'https://registry.npmjs.org' if not set. Set secrets npm_registry_url and npm_registry_authtoken accordingly.
+            pnpm_repository:
+                required: true
+                type: string
+                description: Remote repository containing pnpm repo
+            pnpm_ref:
+                required: true
+                type: string
+                description: Remote repository ref to checkout
+            config_ref:
+                required: true
+                type: string
+                description: Config ref to checkout
+        secrets:
+            acr_registry:
+                required: true
+            acr_username:
+                required: true
+            acr_password:
+                required: true
+            poeditor_api_key:
+                required: false
+                description: necessary if input.run_poeditor is true
+            poeditor_project_id:
+                required: false
+                description: necessary if input.run_poeditor is true
+            npm_registry_url:
+                required: false
+                description: necessary if input.use_custom_registry is true
+            npm_registry_authtoken:
+                required: false
+                description: necessary if input.use_custom_registry is true
+            token:
```

Very cryptic secret name, pls fix.
```suggestion
            pnpm_repository_token:
```